core/src/lib.rs

```diff
@@ -2,6 +2,7 @@
 #![feature(extract_if)]
 #![feature(hash_extract_if)]
 #![feature(proc_macro_span)]
+#![feature(iter_repeat_n)]
```

Comment:
I wonder why is this needed because the latest Rust distribution seems to have stabilized the feature.

Reply:
Hi, without adding it, the installing scallopy using pip will fail. Sorry I'm not very familiar with rust. I think you can also try running `pip install ./etc/scallopy` and see what happened if not adding this feature.

Reply:
Thanks. The checks have passed anyway so I'll move ahead and merge the PR.